tests/events/47-long-output.t

```diff
@@ -62,7 +62,7 @@ purge_suite "${SUITE_NAME}"
 
 # REPEAT: Long STDERR output
 
-init_suite "${TEST_NAME_BASE}" <<'__SUITERC__'
+init_suite "${TEST_NAME_BASE}" <<__SUITERC__
```

Comment:
Unquoted heredoc delimiter lets `$CYLC_DIR` get interpolated by the shell immediately, so that actual directory path to the top level of the repository (not the installed library) gets written to the suite definition, rather than the variable.

This fixes the problem because `$CYLC_DIR` is defined wrongly (for this purpose) in `cylc/flow/__init__.py`... 

`$CYLC_DIR` was always the top level of the repository, not of the Python library...

Reply:
Thanks for the explanation!!! Will try to remember that next time I have a similar problem with another test :+1:

Reply:
(And in the Travis build, CYLC_DIR gets set to TRAVIS_BUILD_DIR (i.e. repository top level, where COPYING is) by the test battery command).